### ISSUE TITLE
Colon character in Fira Code Retina is not symmetrical

### ISSUE THREAD

**Issue by ssoher** [tonsky/FiraCode#765]
As can be seen from the below screenshot, in the colon ':' character the two dots are not of the same size. Is this by design?

![image](https://user-images.githubusercontent.com/4283979/58079608-0e909280-7bba-11e9-9892-73c68e69339d.png)

**Comment by tonsky** [comment]
Probably a hinting issue. What OS are you using? Which editor, font size, font weight, and what display are you using (retina/not)? https://github.com/tonsky/FiraCode/wiki#hinting-issues

**Comment by ssoher** [comment]
Sublime, IntelliJ and emacs
Windows 10 x64 1709
Size 14(px) in IntelliJ 10.5(pt) on sublime and emacs
I didn't set any fontweight and can't see no such option in my editors, so default I guess?
Display is a 40 inch Philips BDM4065UC running at 3840 x 2160 resolution

**Comment by ssoher** [comment]
This problem doesn't seem to happen on any other font size but 14px

**Comment by tonsky** [comment]
Yeah, windows does some aggressive hinting, not much I can do to fix that
